src/deploy/functions/cache/hash.ts

```diff
@@ -24,7 +24,7 @@ export function getEnvironmentVariablesHash(backend: Backend): string {
  * @param pathToGeneratedPackageFile Packaged file contents of functions
  */
 export async function getSourceHash(pathToGeneratedPackageFile: string): Promise<string> {
-  const hash = crypto.createHash("sha256");
+  const hash = crypto.createHash("sha1");
 
   // Hash the contents of the source file
```

Comment:
why is this still reading the archived file - not the hash we generated in prepareFunctionUpload?

Reply:
Ah good catch! The comments are misleading. We aren't hashing the archive anymore, we are hashing the files contained within the archive.

Updated comments!